src/files-card-body.jsx

```diff
@@ -363,7 +363,7 @@ export const FilesCardBody = ({
     });
 
     return (
-        <div id={files_parent_id} ref={folderViewRef}>
+        <div id={files_parent_id} className="fileview-wrapper" ref={folderViewRef}>
```

Comment:
```
# /work/make-checkout-workdir/src/files-card-body.jsx
#   366:64  warning  Prop `ref` must be placed on a new line  react/jsx-max-props-per-line
```

Other then that its good.

Reply:
So is this "correct"? f3404e00fa61c1bde789ec4f17627547f51e0532

(We'll squash all this down when it's time for merging.)